smf/sled-agent/config-rss.toml

```diff
@@ -56,7 +56,7 @@ gz_addresses = []
 [request.service.service_type]
 type = "nexus"
 internal_address = "[fd00:1122:3344:0101::3]:12221"
-external_address = "[fd00:1122:3344:0101::3]:12220"
+external_address = "172.20.15.224:12220"
```

Comment:
Should be `172.20.15.226` per Josh's note

Reply:
Done